front/src/Utils/PathfindingManager.ts

```diff
@@ -8,19 +8,26 @@ export class PathfindingManager {
     private grid: number[][];
     private tileDimensions: { width: number; height: number };
 
-    constructor(scene: Phaser.Scene, collisionsGrid: number[][], tileDimensions: { width: number; height: number }) {
+    constructor(
+        scene: Phaser.Scene,
+        collisionsGrid: number[][],
+        walkingCostGrid: number[][],
+        tileDimensions: { width: number; height: number }
+    ) {
         this.scene = scene;
 
         this.easyStar = new EasyStar.js();
-        this.easyStar.enableDiagonals();
+        // this.easyStar.enableDiagonals();
```

Comment:
Out of curiosity, why did you need to disable diagonals?

1 | 1
-|-
1|1000

I'm assuming in this case, if you go from top right to bottom left, the cost will be small, but you risk walking on the bottom right tile?

Reply:
Currently we will get teleported if we barely touch these fields, so diagonal movement is not an option sadly :(